source/quick-start.txt

```diff
@@ -37,11 +37,62 @@ to check your version of Node and NPM and installation instructions for your
 system, see `downloading and installing Node.js and npm
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
-
 After you verify that you installed the required versions of Node and NPM,
-create a new project.
+create a new project by :ref:`cloning our js-starter repository
+<qs-download-the-repo>` or by :ref:`creating it manually <qs-manual-setup>`.
+
+.. _qs-download-the-repo:
+
+Clone the Project
+~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   This approach requires you to have `Git
+   <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+   installed.
+
+First, clone our `js-starter repository
```

Comment:
```suggestion
Clone the `js-starter repository
```
Suggestion: [Use our/we sparingly](https://docs.mongodb.com/meta/style-guide/writing/write-to-the-user/#id1)

Suggestion: I don't think "first" is necessary. I also think installing git is the first step.

Reply:
Agreeing with "our" -> "the".
I don't think removing the transition word adds additional value, so I will be keeping all of them.